src/js/components/Person/PersonProfileDrawerMainContent.jsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import styled from 'styled-components';
 import { renderLog } from '../../common/utils/logging';
-import QuestionnaireResponsesList from '../Questionnaire/QuestionnaireResponsesList';
 import PersonProfile from './PersonProfile';
```

Comment:
I think we want to leave `import QuestionnaireResponsesList from '../Questionnaire/QuestionnaireResponsesList';` in here but commented out since we want to uncomment `<QuestionnaireResponsesList />` as soon as that component is running again.